gemfileparser/__init__.py

```diff
@@ -25,65 +27,51 @@
 
 
 class GemfileParser(object):
-
-    '''Creates a GemfileParser object to perform operations. '''
+    """
+    Creates a GemfileParser object to perform operations.
+    """
 
     class Dependency(object):
-
-        ''' A class to hold information about a dependency gem.'''
+        """
+        A class to hold information about a dependency gem.
+        """
 
         def __init__(self):
-            self.name = None
+            self.name = ''
```

Comment:
Why use empty strings instead of None?

Reply:
I think the repo which we have forked is some commits behind. Using empty strings here are the original changes in the upstream repo. And may be this is helpful in extracting data from `gemfile` files.